order/supplier/send-order-response/attach-document.md

```diff
@@ -20,34 +16,71 @@ You can attach documents using three methods:
+
+{% page-ref page="./" %}
 
-You can attach the uploaded document using two methods:
+This will NOT create an `OrderDocumentsAttachedBySupplier`activity and NO acknowledge task for the buyer.
 
 {% hint style="warning" %}
-This feature is not available yet.
+The total number of documents is limited to 100 documents per order header and per order line.
+{% endhint %}
+
+### Documents body
+
+* `code` : optional document code or number. The document code should be unique within your company and immutable.
+
+{% hint style="info" %}
+If a document with the same code is already attached to the order or line, the document will be replaced, else it will be appended.
 {% endhint %}
 
+* `revision`: optional document revision \(or version\) code or number
+* `name` : optional document short \(file\) name
+* `description` : optional document description or long name
+* `type`: optional document business type
+* `objectId`: the Tradecloud object identifier which was returned by the Tradecloud object-storage upload
+
 ## Method 2. Attach a document using your company's content server
 
 ### Step 1. Upload to your company's content server
 
 Upload documents to your company's content server and remember the returned document `url`.
 
-### Step 2. Choose an appropriate API to attach the document
+### Step 2. Provide the `url` in the order message using the `/order/order-response` API
 
-Choose an appropriate API to attach the document:
+You can provide the `url` using the `documents` field on either `order` or `lines` level in the order body of the [Send order response](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/api-connector/specs.yaml#/supplier-endpoints/sendOrderResponseBySupplierRoute) endpoint.
 
-{% hint style="warning" %}
-This feature is not available yet.
+### Documents body
+
+* `code` : optional document code or number. The document code should be unique within your company and immutable.
+
+{% hint style="info" %}
+If a document with the same code is already attached to the order or line, the document will be replaced, else it will be appended.
 {% endhint %}
 
+* `revision`: optional document revision \(or version\) code or number
+* `name` : optional document short \(file\) name
+* `description` : optional document description or long name
+* `type`: optional document business type
+* `url`: the document location on the company's content server
+
 ## Method 3. Only provide document meta data
 
 Only provide document meta data like code, revision, name, description and type.
 
-Choose an appropriate API to attach the document:
+You can provide the meta data using the `documents` field on either `order` or `lines` level in the order body of the [Send order response](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/api-connector/specs.yaml#/supplier-endpoints/sendOrderResponseBySupplierRoute) endpoint.
```

Comment:
This too will NOT create an activity or workflow task for the buyer.

Perhaps it's good to mention that if you want this activity / task to happen, the supplier should attach a document through the portal?